src/handlers/playlist.rs

```diff
@@ -62,6 +62,7 @@ pub fn handler(key: Key, app: &mut App) {
       if let (Some(playlists), Some(selected_playlist_index)) =
         (&app.playlists, &app.selected_playlist_index)
       {
+        app.active_playlist_index = Some(selected_playlist_index.to_owned());
```

Comment:
From my look through the codebase, there is no other way in which we enter into the `TableContext::MyPlaylists` context, so this is all that is necessary to track the "active" playlist.